cookbook/how_to_upgrade_to_new_version.rst

```diff
@@ -116,6 +116,10 @@ Uncomment this line.
 
     $ sudo -uwww-data app/console lexik:maintenance:unlock --env prod
 
+    .. note::
+        If There are PHP bytecode cache (e.g. opcache) tools are in use, PHP-FPM (or Apache web server) should be
+        restarted after uprgade to flush cached bytecode from previous installation.
```

Comment:
Edit: Edit: If PHP bytecode cache tools (e.g. opcache) are used, PHP-FPM (or Apache web server) should be restarted after the uprgade to flush cached bytecode from the previous installation.